src/Commands/Modules/Utility/MathCommand.cs

```diff
@@ -23,6 +25,14 @@ public async Task MathCommandAsync([Summary("The LaTeX input."), Remainder] stri
                 return;
             }
             
+            bool canOverride = (Context.User as IGuildUser).GetPermissionLevel(Context) >= PermissionLevel.Moderator;
```

Comment:
imo Guru and up can override this cooldown. It makes more sense for them to have access to this utility than regular users.

Reply:
Changed in f3355500a1ff5f938825f603788a164f9cdf72f0